VSRAD.Syntax/FunctionList/FunctionList.cs

```diff
@@ -161,15 +160,22 @@ public static void TryUpdateFunctionList(ITextSnapshot version, IReadOnlyList<IB
                 ThreadHelper.JoinableTaskFactory.RunAsync(() => Instance.UpdateFunctionListAsync(version, blocks));
         }
 
-        private static FunctionBlock GetFunctionBy(IEnumerable<FunctionBlock> blocks, ITextSnapshotLine line)
+        private static FunctionBlock GetFunctionBy(IEnumerable<FunctionBlock> blocks, SnapshotPoint position)
```

Comment:
This may be simplified to `=> blocks.FirstOrDefault(func => PointInFunction(func, position));`

Reply:
Got it! Thanks for tips.